### ISSUE TITLE
Possibly multiple approve / reject as issue owner/staff

### ISSUE THREAD

**Issue by kuhnchris** [gitcoinco/web#2923]
**Describe the bug**
A common bug on the platform is the multiple "approve/reject" worker messages.
One issue now came into my fangs, and after some dissection was available to figure out the issue (mostly).

**To Reproduce**
After approve/reject reload/refresh the page (either automatically, or due to MetaMask or due to some auto-refresh whatever).
Broadly speaking, if you issue link is https://gitcoin.co/issue/user/repo/issue-nmbr/global-number, append ?mutate_worker_action=approve&worker=someone-who-applied-to-work and you can apply/reject workers as long as you are the owner.
Basically, this code is somewhere broken (weird check for interests)
https://github.com/gitcoinco/web/blob/d8f5bd84db48cefa2a8bad8cb7c7a3e41eeccf46/app/dashboard/views.py#L900-L946

**Expected behavior**
Approval/Rejection once

**Screenshots**
![image](https://user-images.githubusercontent.com/761911/48923108-1fcb4180-eeac-11e8-9e7d-3611b98a25cb.png)
